### DISCUSSION TITLE
A32NX / MCDU is dark

### DISCUSSION THREAD

**Discussion by klakla13** [flybywiresim/aircraft#3630] [Help] [answered]
Hi.
I really enjoy using the A32NX. But if I have other aircraft in the community folder, then my MCDU is dark and cannot be operated. All other functions are OK.
As soon as I remove the planes from AbadStudios, everything works with the MCDU.
I use the AbadStudio aircraft for traffic.
I've also tried all versions of the A32NX and made all the updates.
Thank you for your help.

**Comment by hotshotp** [comment, marked as answer]
Yes, abadstudios are a scam. they are pirated models that are free and are being sold. DO NOT support abadstudios. They cause conflicts with the mod too. I highly suggest you uninstall the aircraft.
